src/components/Materialize/Modal.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <div v-if="isOpen" :id="id" class="modal" :class="computedClasses">
+    <div v-if="isOpen" :id="id" class="modal" :class="computedClasses" data-modal="true">
```

Comment:
This component should die.

Reply:
Like ALL the components in the `Materialize` directory.

Reply:
Why they are not deleted so?

Reply:
Bc I forgot

Reply:
Those components are still used everywhere in the AC. IMHO this is outside the scope of this PR.